xml/art_sle_install_quick.xml

```diff
@@ -915,8 +915,8 @@ disk:
       </para>
       <para>
        The SSH service is enabled by default, but its port (22) is closed in the
-       firewall. Click on <guimenu>open</guimenu> to open it or
-       <guimenu>disable</guimenu> to disable the service. Note that if SSH is
+       firewall. Click on <guimenu>Open</guimenu> to open it or
+       <guimenu>Disable</guimenu> to disable the service. Note that if SSH is
```

Comment:
Turns put both of them were wrong. The GUI items are really lowercase. Reverted in036ab201.